examples/phi3/phi3_vision.py

```diff
@@ -105,11 +106,11 @@ def resave_onnx_model(model_path, target_output_path, pass_config):
 
 
 def run_and_save(config, output_dir, suffix):
-    run_output = olive_run(config)
-    output_node = next(iter(run_output.values())).get_top_ranked_nodes(1)[0]
+    workflow_output = olive_run(config)
+    output_model = workflow_output.get_best_candidate()
     # "model_path" resource can be folder for model with external data
-    model_path = ModelConfig.parse_file_or_obj(output_node.model_config).create_model().model_path
-    pass_config = output_node.pass_run_config
+    model_path = ModelConfig.parse_file_or_obj(output_model._olive_model_config).create_model().model_path
+    pass_config = output_model._model_node.pass_run_config
```

Comment:
nitpick: Can you convert `_olive_model_config` and `_model_node` as properties in `output_model`.